package.json

```diff
@@ -33,33 +33,34 @@
     "test": "jest --runInBand --forceExit",
     "test:watch": "npm run test -- --watch"
   },
+  "peerDependencies": {
```

Comment:
why peerdependencies?

Reply:
Because we need to make sure that this version of ethers is installed in the app that is using the core for everything to work correctly. We could avoid declaring the “peerDependencies” property but i prefer to force the dep installation.

Reply:
and it's not enough to declare it as peer dependency or does it have to be also in the dependency section?

Reply:
Needs to be declared in both "dependencies" section because you want to tell clients that they need to install "ethers" because is a peer dependency and you need to be able to run tests as a "client" will use the core so that is why is defined as a dev dependency as well.